core/src/main/java/com/adobe/aem/guides/wknd/core/models/impl/ImageListImpl.java

```diff
@@ -280,15 +280,15 @@ public String getId() {
         public SimpleImageComponentResource(final Resource resource, final String alt) {
             super(resource);
 
-            // Expose the original fileReference
-            properties.put(PN_FILE_REFERENCE,
-                    resource.getValueMap().get(PN_FILE_REFERENCE));
+            // Copy the properties from the original image
+            properties.putAll(resource.getValueMap());
 
             // Override the decorative configuration attributes
             properties.put(PN_ALT, alt);
             properties.put(Image.PN_IS_DECORATIVE, false);
-            properties.put(Image.PN_DISPLAY_POPUP_TITLE, false);
+            properties.put(Image.PN_DISPLAY_POPUP_TITLE, true);
```

Comment:
Is this being written out as HTML now just to be hidden via CSS?  Don't recall the BEM's off the top of my head to match these up.

```
.cmp-image__title {
      display: none;
  }
```

Reply:
less is more :)